com.beeper.Beeper.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.beeper.Beeper</id>
+  <name>Beeper</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Beeper Team</developer_name>
+  <summary>All your chats in one app. Yes, really.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://www.beeper.com/</url>
+  <url type="help">https://help.beeper.com/</url>
+  <description>
+    <p>We're building the only chat app on Earth</p>
+    <ul>
+      <li>Communicate with people everywhere! Using Matrix Bridge</li>
+      <li>Search across all apps</li>
+      <li>Prioritize your inbox</li>
+      <li>Send scheduled messages</li>
+      <li>One Unified inbox</li>
+    </ul>
+    <p>This wrapper is not officially endorsed by the Beeper Team.</p>
+  </description>
+  <screenshots>
+    <screenshot>
+      <caption>Beeper logged in</caption>
+      <image type="source">https://uploads-ssl.webflow.com/5ffc199ed786423eb2569667/648af12deb905c3acb5fe9c7_screenshot%20no%20background.png</image>
```

Comment:
This links seems like it'd be gone in a few months. Wondering if there's a more stable link. It's also a picture of MacOS not linux.

Reply:
I'm told I should use the upstream link and this is what they use, can't really do anything about that :P

Reply:
I was hoping upstream had a linux screenshot, if not leave it.

Reply:
Sadly upstream has like 1 screenshot on their entire site and it's all MacOS and iOS. No windows or Linux screenshots anywhere. Android only has screenshots in the play store.

Reply:
May I resolve this conversation? I'm not sure how I could "use image from upstream" while "having a Linux screenshot"

Reply:
Yea